### DISCUSSION TITLE
Issue with Profile Scope

### DISCUSSION THREAD

**Discussion by ps-carvalho** [OrchardCMS/OrchardCore#10522] [Q&A]
Hi All.
Im facing a bit of an issue setting following https://www.youtube.com/watch?v=C3-_51u9IlU , Im getting invalid_request This client application is not allowed to use the specified scope. and the scope in question is Profile , 

I have tried to to connect other clients and see to get same issue everytime with Profile , can anyone point me into the right direction?

Thanks in advance for any help :)

**Comment by Skrypt** [comment]
Refer yourself to OpenIddict documentation.

**Reply by ps-carvalho** [reply]
thank you for the help.
This issue is that in the video at 14:56 the presenter shows the client configuration in Orchard and the only scope available is api.
I was assuming the the profile scope was added to the client by default.
To fix my Issue I added the scope profile to scopes and then added the scope to the client.
I don't know what is the expected behaviour but I think that all is good and I need to specify the scope in the client.
